account_invoice_partner_no_autofollow/models/account_move.py

```diff
@@ -0,0 +1,20 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def create(self, values):
+        override = super(AccountMove, self).create(values)
+        if self.env["ir.config_parameter"].sudo().get_param("account.no_auto_follow"):
+            override.message_follower_ids.unlink()
```

Comment:
This is not the right way to do this.
Use "unsubscribe follower" method of mail thread instead.